apps/prairielearn/python/python_helper_sympy_test.py

```diff
@@ -56,6 +56,11 @@ class TestSympy:
     ]
 
     EXPR_PAIRS = [
+        # Test unicode conversion
+        ("m - n", M - N),
+        ("m - （n + m）", -N),
```

Comment:
I'm wondering if using `"m - \uff08n + m\uff09"` might make the intention of this test clearer, since a first skim of the code might not catch the fact that these are not regular brackets. Same for the plus sign in the following line (`"\uff0b"`).

To be honest, although we're leaving in the post-Unicode world where almost anything is assumed to be encoded in UTF-8, I still feel uneasy having non-ASCII characters in source code files.

About the previous line (`m - n`), that one does not have non-ASCII characters, was that intended? Did you mean `"m \u2212 n"` or something similar? You can also replace `n` with `\xf1` (`ñ`), since unidecode also removes accents.

Reply:
That makes sense, I copied and pasted these characters from somewhere, but the intention is clearer using the explicit codes. And yes, I think there was a typo on the first line (which I didn't catch because I assumed the unicode character I added just looked a lot like a minus sign 😅 )